ecosystem/indexer-grpc/indexer-grpc-utils/src/counters.rs

```diff
@@ -210,6 +210,30 @@ pub static TRANSACTION_UNIX_TIMESTAMP: Lazy<GaugeVec> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static IN_MEMORY_CACHE_LATEST_VERSION: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "indexer_grpc_in_memory_cache_latest_version",
+        "Latest version in in-memory cache",
+    )
+    .unwrap()
+});
+
+pub static IN_MEMORY_CURRENT_CACHE_ENTRY_COUNT: Lazy<IntGauge> = Lazy::new(|| {
+    register_int_gauge!(
+        "in_memory_cache_current_total_entry_count",
+        "Current total entry count in in-memory cache",
+    )
+    .unwrap()
+});
+
+pub static REDIS_START_VERSION: Lazy<IntGauge> = Lazy::new(|| {
```

Comment:
Prefer to be consistent about which version we're measuring in the metrics. Let's make this redis latest version to match with all the other places